tests/Unit/Controllers/Api/TraderControllerTest.php

```diff
@@ -54,6 +53,25 @@ protected function setUp()
         // Todo set some of the pended_at times to yesterday.
     }
 
+    /**
+     * Tests the for the api.trader.market route and controller.
+     */
+    public function testShowMarket()
```

Comment:
Awesome. We aren;t doing complete coverage on this project but if we were - you'd want - can't see another trader's market etc.